### PR TITLE
Add filter input to departments picker

### PR DESCRIPTION
**Pull request by bastienguillon**
Add filter input to departments picker for the api

### CODE REVIEW

**Review by strousset — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1578

File: packages/ng/department/src/lib/service/department-v3.service.ts [hunk 1]
```diff
@@ -31,9 +33,11 @@ export class LuDepartmentV3Service extends LuApiV3Service<ILuDepartment> impleme
 	getTrees() {
 		let call: Observable<ILuApiResponse<IApiDepartment>>;
 		if (this._appInstanceId && this._operations?.length) {
-			call = this._http.get<ILuApiResponse<IApiDepartment>>(`/api/v3/departments/scopedtree?fields=id,name&appInstanceId=${this._appInstanceId}&operations=${this._operations.join(',')}`);
+			call = this._http.get<ILuApiResponse<IApiDepartment>>(
+				`/api/v3/departments/scopedtree?fields=id,name&${[`appInstanceId=${this.appInstanceId}`, `operations=${this._operations.join(',')}`, this._filters].filter((f) => !!f).join('&')}`,
```
Comment (shajz) [inline packages/ng/department/src/lib/service/department-v3.service.ts:37]:
```suggestion
				`/api/v3/departments/scopedtree?fields=id,name&${[`appInstanceId=${this.appInstanceId}`, `operations=${this._operations.join(',')}`, this._filters.join(',')].filter((f) => !!f).join('&')}`,
```

**Review by shajz — CHANGES_REQUESTED** [review]

File: packages/ng/department/src/lib/service/department-v3.service.ts (hunk 1 above)
Comment (bastienguillon) [reply packages/ng/department/src/lib/service/department-v3.service.ts]: Bien vu !

**Review by shajz — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1578

**Pull request merged**